src/app/clusters/basic-information/basic-information.cpp

```diff
@@ -79,18 +93,18 @@ CHIP_ERROR EncodeStringOnSuccess(CHIP_ERROR status, AttributeValueEncoder & enco
 
 CHIP_ERROR BasicAttrAccess::Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder)
 {
-    if (aPath.mClusterId != BasicInformation::Id)
-    {
-        // We shouldn't have been called at all.
-        return CHIP_ERROR_INVALID_ARGUMENT;
-    }
-
-    CHIP_ERROR status = CHIP_NO_ERROR;
+    CHIP_ERROR status         = CHIP_NO_ERROR;
+    auto * deviceInfoProvider = GetDeviceInstanceInfoProvider();
+    auto & configManager      = ConfigurationMgr();
 
     switch (aPath.mAttributeId)
     {
+    case ClusterRevision::Id:
```

Comment:
Add it in the JSON as "implemented via attribute access interface) so we stop allocating space for it in the attr store?

Reply:
Will do in a follow-up because I need to regen 17 million files :)